portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/themesinfrastructure/Theme.testcase

```diff
@@ -104,6 +104,11 @@ definition {
 
 		SikuliAssertElementPresent(locator1 = "Theme#TEST_THEME_HOME_PAGE_PNG");
 
+		AssertCssValue(
```

Comment:
Could we extract this assertion elsewhere so that we can always do `AssertDefaultBodyBackgroundColorCssValue` or something like that? 

That way, if the background color changes (not unheard of), we would only need to change it in one place rather than in 4 as in this PR.

Alternatively, simply extract `rgba(255, 165, 0, 1)` to somewhere as a constant that you can import so that we only need to change that instead of 4 values.

Reply:
Thanks for this review! I'll look into making this assertion more easily maintainable.